provisioning/transport/http/src/HttpAuthStrategyX509.cs

```diff
@@ -20,13 +22,22 @@ public HttpAuthStrategyX509(SecurityProviderX509 security)
             _security = security;
         }
 
-        public override DeviceProvisioningServiceRuntimeClient CreateClient(Uri uri)
+        public override DeviceProvisioningServiceRuntimeClient CreateClient(Uri uri, IWebProxy proxy)
         {
             _certificate = _security.GetAuthenticationCertificate();
 
+            HttpClientHandler httpClientHandler = new HttpClientHandler();
```

Comment:
This now seems common code - please extract in the common portion, leaving only TPM/X509 particular code in these classes.